demo/grid-styles.js

```diff
@@ -1,6 +1,8 @@
-<link rel="import" href="../../polymer/polymer.html">
+// import '../node_modules/@polymer/polymer/polymer.js';
+const $_documentContainer = document.createElement('div');
+$_documentContainer.setAttribute('style', 'display: none;');
 
-<dom-module id="grid-styles">
+$_documentContainer.innerHTML = `<dom-module id="grid-styles">
```

Comment:
What the hell is that ?

Reply:
Okay, I had a look. It looks like we should get rid of the styles modules. Let's go back to the basics. Create a simple stylesheet and import it as usual CSS.